services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/shared.ts

```diff
@@ -206,6 +228,8 @@ const TOPIC = 'topic'
 
 export const sortFunctions = {
   [DEFAULT]: (activities) => activities,
+  [GRADE_LEVEL_ASCENDING]: (activities) => numberAscendingSort(activities, 'minimum_grade_level'),
+  [GRADE_LEVEL_DESCENDING]: (activities) => numberDescendingSort(activities, 'minimum_grade_level'),
```

Comment:
I don't know if this is actually better, but rather than writing another `sort` algorithm, would it make sense to just `reverse` the ascending sort like this: `[GRADE_LEVEL_DESCENDING]: (activities) => numberAscendingSort(activities, 'minimum_grade_level').reverse()`?

Reply:
Yeah, I get your point. I think the way it's currently written will always sort "null" values to the back, whereas just reversing the sort will have them come first in descending order. If there were never null values, I think your version would be better.